src/views/CreateTransformer.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <main v-if="formState === 0" @keydown.enter="nextStep">
+    <ViewHeader title="How should the new transformer be called?" :backLink="{ link: { name: 'transformers' } }"/>
+    <Input :value.sync="newTransformer.name" name="Name" placeholder="Filter for today" :autofocus="true"/>
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 1">
+    <ViewHeader title="Which resource type should be transformed?" :backLink="{ onClick: previousStep }" />
+    <ListSection :list="resourceTypesList"/>
+  </main>
+
+  <main v-else-if="formState === 2">
+    <ViewHeader title="Please choose the type of the transformer." :backLink="{ onClick: previousStep }" />
+    <Select
+          :value.sync="newTransformer.transformerType"
+          name="Type of the transformer"
+          :required="true"
+          :options="transformerTypeOptions"
+        />
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 3">
+    <ViewHeader title="Please insert the functionality of the transformer" :backLink="{ onClick: previousStep }" />
+    <ListSection
+      :title="`Attributes of ${newTransformer.resourceType.name}`"
+      :list="resourceTypeAttributeList"
+    />
+    <br>
+    <h1>below you can enter the code of your transformer</h1>
+    <br>
+    <TextArea
+      :firstString="`instancesOf${newTransformer.resourceType.name}.${newTransformer.transformerType}( (instance) => {`"
+      secondString='});'
+      :value.sync="newTransformer.body"
+      placeholder="return (instances.getAttribute(age) > 18);"
```

Comment:
- [ ] should be `instance` instead of `instances`
 - [ ] `age` must be provided as string → should be `'age'`
 - [ ] it would be nice to also see the `setAttribute` method with an example

Reply:
included all three in the example, could you please check it again for functionality?
The ugly identation is because the textarea takes the identation as part of the string.

Reply:
🎉 that's pretty cool!